### PR TITLE
add esg wp3 first year news

### PR DESCRIPTION
**Pull request by mtangaro**
First year achievements blog post for EuroScienceGateway WP3.

### CODE REVIEW

File: content/news/2023-12-22-esg-wp3-update/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+* IISAS (Slovakia) - SK01
+* CNRS - GenOuest (FR) - FR01 
+* CESNET (Czech Republic) - CZ01
+* EGI and INFN - EGI01
+* BSC (ES) - ES01
+* TUBITAK ULAKBIM setup is almost completed
+
+Moreover, the updated Open Infrastructure allows to deploy full fledged usegalaxy.eu replica servers, thus allowing to instantiate new usegalaxy services easily, but also providing a robust framework for maintaining and updating running instances. We also started documenting the whole procedure, which has been successfully used to deploy a prototype of the Italian UseGalaxy server ([temporary repository](https://usegalaxy-it.github.io/documentation/)).
+
+Currently, 7 UseGalaxy endpoints are available: EU (ALU-FR), Be (VIB), Fr (CNRS), Es (BSC-CNS), No (ELIXIR-NO), Cz (CESNET) and It (CNR). In this case the activities focused on the management and maintenance of the instances:
```
Comment (bgruening) [inline content/news/2023-12-22-esg-wp3-update/index.md:68]: Do we have URLs to all of those?

File: content/news/2023-12-22-esg-wp3-update/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+* Align to Galaxy 23.0+ release
```
Comment (bgruening) [inline content/news/2023-12-22-esg-wp3-update/index.md:69]: Is align = Update to 23.0 release? Are we not already at 23.1 or is that meant by 23.0+?

File: content/news/2023-12-22-esg-wp3-update/index.md (hunk 1 above)
Comment (mtangaro) [reply content/news/2023-12-22-esg-wp3-update/index.md:68]: Sure, I'll add

File: content/news/2023-12-22-esg-wp3-update/index.md (hunk 2 above)
Comment (mtangaro) [reply content/news/2023-12-22-esg-wp3-update/index.md:69]: Currently, only EU is at 23.1. The others have updated to 23.0 recently.

File: content/news/2023-12-22-esg-wp3-update/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+* Switch to Total Perspective Vortex dynamic job destination tool ([pre-print](https://arxiv.org/abs/2312.02060))
+* Pulsar endpoints integration
+* OS image and PostgreSQL updates.
+
+Finally, in order to enable other workflow engines to access and leverage the European Pulsar Network, we are going to implement support for the GA4GH Task Execution Service, as a proof of concept, allowing other services to submit jobs via TES to Pulsar and therefore to the European Pulsar Network. The Task Execution Service (TES) API is an effort to define a standardized schema and API for describing batch execution tasks. A task defines a set of input files, a set of containers and commands to run, a set of output files, and some logging and metadata.
```
Comment (bgruening) [inline content/news/2023-12-22-esg-wp3-update/index.md:74]:
```suggestion
Finally, in order to enable other workflow engines to access and leverage the a Pulsar Network, we are going to implement support for the GA4GH Task Execution Service, as a proof of concept, allowing other services to submit jobs via TES to Pulsar. The Task Execution Service (TES) API is an effort to define a standardized schema and API for describing batch execution tasks. A task defines a set of input files, a set of containers and commands to run, a set of output files, and some logging and metadata.
```

File: content/news/2023-12-22-esg-wp3-update/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+---
+title: "EuroScienceGateway Work Package 3: first year achievements"
+tease: ""
+hide_tease: true
+authors: "Marco Antonio Tangaro, Federico Zambelli"
```
Comment (bgruening) [inline content/news/2023-12-22-esg-wp3-update/index.md:5]: I think this needs to be a list.

**Pull request merged**